tests/Kros.AspNetCore.Tests/Extensions/HttpResponseMessageExtensionsShould.cs

```diff
@@ -0,0 +1,77 @@
+﻿using FluentAssertions;
+using Kros.AspNetCore.Exceptions;
+using Kros.AspNetCore.Extensions;
+using System;
+using System.Net;
+using System.Net.Http;
+using Xunit;
+
+namespace Kros.AspNetCore.Tests.Extensions
+{
+    public class HttpResponseMessageExtensionsShould
+    {
+        [Fact]
```

Comment:
Nebolo by lepšie použiť `[Theory]` miesto `[Fact]`. Bolo by menej testov a ľahšie by sa pridávali ďalšie scenáre.

Reply:
Nie je to take jednoduche, ta assertovacia metoda je genericka, nema nejaky variant kde sa posle typ ako parameter a genericka theory sa urobit neda. Potom by som musel tu genericku metodu volat cez reflexiu, a to asi v testoch nechceme